protocol-definitions/MC.yaml

```diff
@@ -845,3 +845,17 @@ methods:
       retryable: false
       partitionIdentifier: -1
     response: {}
+  - id: 33
+    name: resetAgeStatistics
```

Comment:
Maybe rename it to `resetQueueAgeStatistics` as this is under the new namespace that is not related to the queue?